blueprints/frontend.py

```diff
@@ -123,3 +123,13 @@ async def register_post():
 
     # user has successfully registered.
     return await render_template('verify.html')
+
+""" rules """
+@frontend.route('/rules') # GET
+async def rules():
+    return await render_template('rules.html')
+
+""" discord redirect """
+@frontend.route('/discord') # GET
+async def discord():
+    return redirect("https://discord.com/invite/Y5uPvcNpD9")
```

Comment:
i will make some discord link config soon

Reply:
its better to leave it there for now; you can easily change it whenever it's done (it's just there because 404 bad)